src/Play.js

```diff
@@ -1,15 +1,19 @@
 import React, {useState, useEffect} from 'react';
 import Firestore from './services/firestore.js';
+import Authentication from './services/authentication.js';
+import firebaseConfig from './firebase'; // Must be present to initialize database.
```

Comment:
You don’t need to have firebaseConfig in Play.js

Reply:
Fixed.